### PR TITLE
Add this repository to ArtifactHub and fix maintainers.

### PR DESCRIPTION
**Pull request by desaintmartin**
Fixes https://github.com/oauth2-proxy/manifests/issues/24

@tlawrie I'd need your email in order to add you to the ArtifactHub org I've just created.

Note: after merge, we'll need to copy this file in gh-pages branch to be [verified](https://artifacthub.io/docs/topics/repositories/#verified-publisher)

### CODE REVIEW

**Comment by desaintmartin** [comment]
@tlawrie @NickMeves @JoelSpeed, if you want to manage the ArtifactHub org, please create an account there and I'll add you as owner.

**Comment by desaintmartin** [comment]
Here is our chart: https://artifacthub.io/packages/helm/oauth2-proxy/oauth2-proxy \o/

File: helm/oauth2-proxy/Chart.yaml [hunk 1]
```diff
@@ -22,8 +22,12 @@ sources:
 - https://github.com/oauth2-proxy/oauth2-proxy
 - https://github.com/oauth2-proxy/manifests
 maintainers:
-- email: cedric@desaintmartin.fr
-  name: desaintmartin
+- name: desaintmartin
+  email: cedric@desaintmartin.fr
 - name: tlawrie
+- name: NickMeves
+  email: nick.meves@greenhouse.io
```
Comment (JoelSpeed) [inline helm/oauth2-proxy/Chart.yaml:29]: Nick is no longer with greenhouse, so will need to update this one

@NickMeves Which email do you want to use here?

File: artifacthub-repo.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Artifact Hub repository metadata file
+repositoryID: 84a61f0d-c836-4a44-91ab-9c60748748df
+owners:
+- name: desaintmartin
+  email: cedric@desaintmartin.fr
+- name:  JoelSpeed
+  email: joel.speed@hotmail.co.uk
+- name: NickMeves
+  email: nick.meves@greenhouse.io
```
Comment (JoelSpeed) [inline artifacthub-repo.yml:9]: Nicks email will need updating, see other comment

**Review by JoelSpeed — COMMENTED** [review]
Thanks for setting this up! 

I've created an account via the GitHub SSO, so I assume it should link up to my email as in the files

Just one little thing to fix, but we will need @NickMeves to comment before we can fix it

File: helm/oauth2-proxy/Chart.yaml (hunk 1 above)
Comment (NickMeves) [reply helm/oauth2-proxy/Chart.yaml:29]: `nicholas.meves@gmail.com`

**Review by JoelSpeed — APPROVED** [review]
LGTM, feel free to merge if you're happy this is ready

**Comment by desaintmartin** [comment]
Let's see if artifact hubs like the fact that one user lacks email.

**Pull request merged**